labs/RivePlayer/samples/RivePlayer.Samples/RivePlayer.md

```diff
@@ -17,6 +17,181 @@ subcategory: Input
+		<Rive:TriggerInput Target="trigSuccess"
+						   x:Name="SuccessTriggerInput" />
+</Rive:RivePlayer>
+<Button Content="Success">
+	  <Interactivity:Interaction.Behaviors>
+			  <Interactions:EventTriggerBehavior EventName="Click">
+						<Interactions:CallMethodAction MethodName="Fire"
+												       TargetObject="{x:Bind SuccessTriggerInput}" />
+			  </Interactions:EventTriggerBehavior>
+      </Interactivity:Interaction.Behaviors>
+</Button>
+```
+
+In the example above, we nest the `Rive:TriggerInput` strictly under the `Rive:RivePlayer` markup.
+
+To create a reference to the trigger input named `trigSuccess` in the state machine of this Rive file, we set the `Target` property to that input name. Additionally, the `x:Name` property defines the name for the class so that the button knows what to bind the click event to to "fire" the trigger.
+
+When the button is clicked, it will call into the `SuccessTriggerInput` input class and invoke the `Fire`  method on the `trigSuccess` Rive trigger input where the state machine will respond accordingly.
+
+Additionally in this example, we use the [EventTriggerBehavior](https://github.com/Microsoft/XamlBehaviors/wiki/EventTriggerBehavior) and [CallMethodAction](https://github.com/Microsoft/XamlBehaviors/wiki/CallMethodAction) APIs from the [XAML Behaviors](https://github.com/Microsoft/XamlBehaviors/wiki) dependency. You can include the dependency in that example with the following snippet below in a `Dependencies.props` file:
+
+```xml
+<!-- WinUI 2 / UWP -->
+<ItemGroup Condition="'$(IsUwp)' == 'true'">
+    <PackageReference Include="Microsoft.Toolkit.Uwp.UI.Behaviors"
+                      Version="7.1.2"/>
+</ItemGroup>
+
+<!-- WinUI 2 / Uno -->
+<ItemGroup Condition="'$(IsUno)' == 'true' AND '$(WinUIMajorVersion)' == '2'">
+    <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Behaviors"
+                      Version="7.1.11"/>
+</ItemGroup>
+
+<!-- WinUI 3 / WinAppSdk -->
+<ItemGroup Condition="'$(IsWinAppSdk)' == 'true'">
+    <PackageReference Include="CommunityToolkit.WinUI.UI.Behaviors" Version="7.1.2"/>
+</ItemGroup>
+
+<!-- WinUI 3 / Uno -->
+<ItemGroup Condition="'$(IsUno)' == 'true' AND '$(WinUIMajorVersion)' == '3'">
+    <PackageReference Include="Uno.CommunityToolkit.WinUI.UI.Behaviors"
+                      Version="7.1.100-dev.15.g12261e2626"/>
+</ItemGroup>
+```
+
+And to include the `Interactions` and `Interactivity` namespace and Behaviors APIs, add the following in your XAML:
+```xml
+<Page
+    ...
+    xmlns:Interactions="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:Interactivity="using:Microsoft.Xaml.Interactivity"
+>
+```
+
+## Other Notes
+
+Currently, this runtime uses [SKSwapChainPanel](https://docs.microsoft.com/en-us/dotnet/api/skiasharp.views.uwp.skswapchainpanel?view=skiasharp-views-2.88) from SkiaSharp to do GPU rendering. While you may notice slower frame rates drawing Rives, we are actively working on improving the rendering performance.
```

Comment:
We only use SKSwapChainPanel in UWP. WinAppSdk and Uno use SKXamlCanvas, which is even slower. How about:

This runtime is built on SkiaSharp, which is still being optimized for all platforms. While you may notice slower frame rates drawing Rives, particularly on WinAppSdk, we are actively working (with the SkiaSharp team? Or should we not say that??) to improve the rendering performance.

Reply:
We can just say we're actively working to improve it I think